2024-08-10 05:32:39.539 INFO page_ 1:conftest.py:34: new page opened: [<Page url='about:blank'>]
2024-08-10 05:32:39.548 DEBUG tests_api.utils.logger:logger.py:23: > POST https://dev.agentum.pro/backend/api/auth/get-enter-code {'method': 'POST', 'url': 'https://dev.agentum.pro/backend/api/auth/get-enter-code', 'json': {'phone': ''}, 'headers': {'Accept': 'application/json'}, 'params': None, 'files': None}
2024-08-10 05:32:39.549 DEBUG urllib3.connectionpool:connectionpool.py:1048: Starting new HTTPS connection (1): dev.agentum.pro:443
2024-08-10 05:32:39.644 DEBUG urllib3.connectionpool:connectionpool.py:546: https://dev.agentum.pro:443 "POST /backend/api/auth/get-enter-code HTTP/1.1" 502 157
2024-08-10 05:32:39.645 DEBUG tests_api.utils.logger:logger.py:27: < 502 https://dev.agentum.pro/backend/api/auth/get-enter-code <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.18.0</center>
</body>
</html>